app/serializers/user_serializer.rb

```diff
@@ -31,7 +31,7 @@ def services
       BackendServices::EDUCATION_BENEFITS
     ]
     service_list += [BackendServices::RX, BackendServices::MESSAGING] if object.can_access_mhv?
-    service_list << BackendServices::DISABILITY_BENEFITS if object.can_access_evss?
+    service_list << BackendServices::EVSS_BENEFITS if object.can_access_evss?
```

Comment:
I think we should rename the constant but leave its value the same as before, then update the front end to use either the old or new value then update the backend to send a new value.